apps/api/src/app/shared/framework/idempotency.interceptor.ts

```diff
@@ -48,8 +45,9 @@ export class IdempotencyInterceptor implements NestInterceptor {
     const user = this.getReqUser(context);
     const { organizationId, environmentId, _id } = user;
 
-    const isEnabled = await this.getIsApiIdempotencyEnabled.execute(
-      FeatureFlagCommand.create({
+    const isEnabled = await this.getFeatureFlag.execute(
+      GetFeatureFlagCommand.create({
+        key: FeatureFlagsKeysEnum.IS_API_IDEMPOTENCY_ENABLED,
```

Comment:
Using the new refactored `GetFeatureFlag` and `GetFeatureFlagCommand` in the backend. Only the `FeatureFlagsKeysEnum` is needed in addition, and we don't need a new function for each feature now.

Reply:
Love it, I thought to look at this part myself someday, so thanks for refactoring it! 👑

In addition, I would like to transfer as much of our logic as possible into a shared library so that both the client and the backend can reuse it.